src/main/java/com/test/sb/service/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.test.sb.service;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
```

Comment:
```suggestion
public class FileReaderServiceImpl implements FileReaderService {
```
can it read only from csv files or from any files?

Reply:
Agree, not only from csv